pegasus/sites.v3/code.org/public/prize.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'CS Leaders Prize - $1 Million for U.S. Schools'
+theme: responsive
+social:
+  "og:description": "Tell us how your school will expand computer science, and you could win $10,000 to make it happen!"
+  "og:image": "/images/marketing/hoc2022-cs-leaders-prize-opengraph.png"
+  "og:image:width": 1200
+  "og:image:height": 630
+---
+
+<link href="/css/generated/cs-leaders-prize.css" rel="stylesheet">
+
+# CS Leaders Prize
+
+<header>
+  <div>
+    <h2>Expand computer science at your school, win $10,000</h2>
+    <p>To help more students access computer science, especially students from underserved communities, we're giving away $1 million in awards — that's right, a MILLION dollars. In partnership with DonorsChoose.org, this CS Education Week one middle and high school in every U.S. state and D.C. will be awarded the CS Leaders Prize of $10,000 to help establish computer science education for their students.</p>
+    <a href="#form"><button>Enter to win</button></a>
+  </div>
+  <div>
+    <img src="/images/marketing/hoc2022-cs-leaders-prize-money-bag.png" alt="Icon of a green bag with a purple dollar sign on it surrounded by gold coins">
+  </div>
+</header>
+
+
+## To be eligible a school must commit to:
+
+<ul>
+  <li>Establishing a <strong><em>new</em></strong> <a href="/educate/csp">Computer Science Principles</a>, <a href="/educate/csa">Computer Science A</a>, or <a href="/educate/csd">Computer Science Discoveries</a> course using Code.org.</li>
+  <li>Describe how you plan to continue expanding access to CS in your school and/or district.</li>
+  <li>Provide a short statement of support from district leadership for expanding CS opportunities as a district priority.</li>
+  <li>Organize a school-wide <a href="https://hourofcode.com/">Hour of Code</a> or other event/assembly for <a href="https://www.csedweek.org/">CSEdWeek</a> to inspire students to explore CS and to encourage diverse participation by historically underrepresented groups.</li>
+</ul>
+
+In addition to the $10,000 awarded to the school to fund a project(s) of the school's creation through DonorsChoose, the implementing teacher(s) in each winning school will receive a no-cost scholarship to attend Code.org's [Professional Learning Program](/educate/professional-learning/middle-high).
```

Comment:
I realize I'm not supposed to be copy-editing, but the `a project(s)` phrase is inconsistent –– `(a) project(s)` or `projects` or `a project` would at least not raise my this-is-wrong flag (but it might not be your decision, Kelby)